src/components/flat-table/flat-table.test.js

```diff
@@ -3199,12 +3217,17 @@ context("Tests for Flat Table component", () => {
         .should("have.css", "background-color", "rgb(25, 71, 94)");
     });
 
-    it("should render Flat Table Header and Cells with custom vertical header and body borders", () => {
+    it.only("should render Flat Table Header and Cells with custom vertical header and body borders", () => {
```

Comment:
it("should render Flat Table Header and Cells with custom vertical header and body borders", () => {